nearlib/account.js

```diff
@@ -151,5 +151,40 @@ class Account {
     async viewAccount (accountId) {
         return await this.nearClient.viewAccount(accountId);
     }
+
+    /**
+     * Returns full details for a given account.
+     * @param {string} accountId id of the account to look up 
+     */
+    async getAccountDetails(accountId) {
+        accountId; // make linter happy
+        // Hardcoded for now
+        const result = {
+            authorizedApps : [
+                {
+                    contractId: 'cryptomonsters',
+                    name: 'Monsters',
+                    balance: 10,
+                },
+                {
+                    contractId: 'pay_mo',
+                    name: 'Paymo',
+                    balance: 20 
+                }
+            ],
+            transactions : [
+                {
+                    id: '24359uy1dfsgwr',
+                    transactionType: 'TRANSFER_TOKENS', // to be converted to a description string on front end for easier localization
+                    transactionDetails: {
```

Comment:
I'd prefer we do not wrap stuff into nested objects without much need. It makes checking for presence of values more annoying, i.e. you need to check that `transactionDetails` is set before checking it's fields.

Reply:
There is value in doing this specifically to transaction details, because different types will have diff details. We are most likely going to need code to validate that the details provided match the type anyways, and it's going to be nicer when details are defined in their own object.

Reply:
It might be nicer to define them separately in a statically typed language, cause otherwise you'll have to put unrelated stuff together in one class, etc. In JS it doesn't really matter.

Somewhat related: probably makes sense to add examples for different transaction types.

Reply:
added a few more types of examples. I flattened the structure - I still prefer the nested object method, but we need to move forward so whatever.